e2e/testdata/otel-collector/otel-cm-monarch-rejected-labels.yaml

```diff
@@ -72,8 +70,6 @@ data:
           # will not break this ingestion pipeline
           create_service_timeseries: true
           service_resource_labels: false
-        retry_on_failure:
```

Comment:
do we need to update any docs for this change?

Reply:
We don't document the `retry_on_failure` field in ACM guide pages, but this field is configurable via a custom ConfigMap.
That being said, it is a breaking change if we want to include it into the current patch release.
But if we don't cherrypick the change, there will be fixable vulnerabilities in the current version.

I'm inclined to document the change in a release note and have the CVE fix in 1.16.2. @janetkuo WDYT?